scripts/visualize_signals.py

```
@@ -0,0 +1,294 @@
+            "hedge_pressure": "#4ECDC4",
+            "basis_pressure": "#45B7D1",
+            "confidence": "#96CEB4",
+            "stability": "#FFEAA7",
+            "health": "#DDA0DD",
+        }
+
+    def check_api_health(self) -> bool:
+        """Check if the API is running"""
+        try:
+            response = requests.get(f"{self.api_url}/api/v1/health", timeout=5)
+            return response.status_code == 200
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)